packages/choreography-lint/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "eslint-plugin-choreography-lint",
+  "type": "commonjs",
+  "version": "1.0.0",
+  "description": "Custom ESLint plugin for choreographic programming ",
+  "main": "dist/index.js",
```

Comment:
I see no `dist` or a build command. We should write rules in TypeScript, set up tsc to generate `dist` (see other directories).